examples/alternate-receive/alternate_receive.md

```diff
@@ -0,0 +1,7 @@
+# Alternate receive
+
+The alternate receive action can be used to receive values from several workers. It operates by waiting until it encounters a non-error message, a panic termination status on a closed channel, or the closure of all channels. Alternate receive action sets the first non-error value it encounters as the outcome.
```

Comment:
One more thing. `alternate-receive` can be used to receive values from sends within the same worker as well. 
May be
```suggestion
The alternate receive action can be used to receive values from mutiple send actions. It operates by waiting until it encounters a non-error message, a panic termination status on a closed channel, or the closure of all channels. Alternate receive action sets the first non-error value it encounters as the outcome.
```